easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-CrayGNU-20.08-EGL-python3.eb

```diff
@@ -0,0 +1,79 @@
+# CrayGNU version by Jean Favre (CSCS)
+
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.8.1'
+versionsuffix = '-EGL-python%(pymajver)s'
+
+homepage = "http://www.paraview.org"
+description = "ParaView is a scientific parallel visualizer."
+
+toolchain = {'name': 'CrayGNU', 'version': '20.08'}
+toolchainopts = {'pic': True, 'usempi': True, 'verbose': False}
+
+local_download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % local_download_suffix]
+sources = ["ParaView-v%(version)s.tar.gz"]
+
+dependencies = [
+#    ('h5py', '2.8.0', '-python3-serial'),
```

Comment:
```suggestion
```

Reply:
Thanks @jfavre: if the `h5py` dependency is not needed any longer, I suggest to remove it from the list

Reply:
I have temporarily removed the h5py in both ParaView recipes in order to move forward with testing the app. I plan to add it at a later time. It is an optional component.